### ISSUE TITLE
Component Runner - Cypress Testing Library error reporting appears differently in the CT runner vs normal E2E runner

### ISSUE THREAD

**Issue by JasonFairchild** [cypress-io/cypress#16120]
### Current behavior
When I do a findByRole query in the normal runner, it looks like this from the command log:
![image](https://user-images.githubusercontent.com/56280950/115587675-2a5fb000-a28b-11eb-9f28-748e492f37f1.png)

When I do the same type of query from the component test runner I don't see the error in the command log:
![image](https://user-images.githubusercontent.com/56280950/115589461-1ae16680-a28d-11eb-90aa-0cb5c7539c2d.png)

### Desired behavior
Not a huge deal, but I prefer the error to be in the command log. I use the list of roles/names all the time to help determine what to query by.

### Test code to reproduce
I'm confident this could be reproduced with any query using the Cypress testing library along with the component runner.

### Versions
I don't imagine these version matter much except for what is required for the CT runner.
I'm on Windows 10
Chrome 89
The normal runner I am using is Cypress v6.8.0
The CT runner I am using is Cypress v7.1.0

**Comment by jennifer-shehane** [comment]
I'm not experiencing this when I use the cypress-testing-library within the component testing runner. I do see the error print. 

There's likely something else involved in your repo/config that is affecting the error display.

Can you provide a reproducible example? I pulled down the https://github.com/elevatebart/calc repo and added testing-library, added `cy.findByRole('button', { name: /Jackie Chan/i }).click()` to the test. 

<img width="587" alt="Screen Shot 2021-04-21 at 2 40 53 PM" src="https://user-images.githubusercontent.com/1271364/115611497-c4d1ea80-a2af-11eb-968a-94e0630a90ba.png">

**Comment by JasonFairchild** [comment]
Ah snap. @jennifer-shehane were you using Windows by chance? Thanks for trying it out though. I have another issue I'm trying to figure out. Then I'll try this again and see if I figure out more.

**Comment by jennifer-shehane** [comment]
No, I'm using a Mac.

**Comment by JasonFairchild** [comment]
Well, I wouldn't have a clue why, but I wouldn't be surprised if it was windows related. I've had several issues lately that turned out to be windows related anyway, especially in connection to component testing.

The reason I haven't followed up on this for a while though is because I'm having other issues with Cypress and the plugin Cypress testing library. So trying to figure those out first to see if it makes any difference.

**Comment by JasonFairchild** [comment]
Ok, I got all my known issues with Cypress and the testing library plugin figured out except this one. I still observe the behavior where the E2E tester places the testing library error in the command log but the CT runner puts it in the dev console.

@elevatebart could I ask you a favor of trying this out on Windows? I just ask as I believe you have relatively easy access to do so.

The only other info I can think of on this is that I know the old CT runner (Cypress 6.8, which I understand is basically just the current E2E runner running component tests) would still put the testing library error in the command log. So I think it is perhaps slightly less likely it is just our webpack config or something like that, since that shouldn't have changed. But I'm open to anything, considering Jennifer cannot reproduce it.

**Comment by elevatebart** [comment]
@JasonFairchild thank you for all the details.
I'll try this on windows on Monday.

Perhaps something obvious will appear.

**Comment by elevatebart** [comment]
@JasonFairchild I just tested making errors appear on [this repo](https://github.com/elevatebart/calc) 
On windows, I got the extra info. It is probably something else.
If you find you have time to fork it and reproduce your bug on this Repo (by tweaking to look more like yours) I would appreciate it.

Thank you

**Comment by JasonFairchild** [comment]
Alright, thanks for trying it out. I'll try that sometime.

**Comment by lmiller1990** [comment]
I had a look at also was unable to reproduce this. 🤔 

![image](https://user-images.githubusercontent.com/19196536/117406170-0b7e4080-af50-11eb-957c-6c2cec049cfd.png)

**Comment by JasonFairchild** [comment]
Sorry for the long time for response. I cloned the calc repo and I could not reproduce the issue there either. So I suppose it must be something specific to that repository I was producing it with. I have no idea what though. On top of that, I was removed from the project, so I am no longer working on component testing there, unfortunately. I have at least verified today that I can still reproduce it there, even with Cypress 7.2. So while I know this issue can happen and I'm sure it could be figured out and improved, there doesn't seem to be much reason to put a lot of effort into this, especially if others do not report it. Not a severe issue in the first place and I don't think I have the time or motivation to pursue it heavily myself. Though if someone else is really interested in looking into it, I'm happy to share more info about the configuration of that project.
